### ISSUE TITLE
Add logging to the workflow job that updates version numbers and incorrect regular expression

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#823]
As title. When releasing v3.3, the workflow failed to update the version numbers (or else I assumed version numbers would already be updated) when it was triggered by the tag creation for that release.

**Comment by Jaifroid** [comment]
Actually, this is already being done. I'm changing the title to reflect the need to add some logging.

**Comment by Jaifroid** [comment]
@mossroy I've discovered the error that caused the workflow to fail with the tag-pattern. You were right that it is related to the fact that we had to edit the standard tag-pattern, but I hadn't understood the importance of the group (parenthesis) in the pattern matching regex for this particular Action. To explain:

* Standard/default tag pattern is: `/^v*([0-9.]+)$/`
* The tag-pattern I had defined for Kiwix JS was (removing the `v` and replacing with a number): `/^[0-9]([0-9.]+).*$/`
* I use the top one as-is in KJSW, so I never detected the problem with the second one
* Documentation says: "Only git tags matching this regexp will trigger a build+push to the corresponding docker tag.
If not specifying a group, whole git tag is used as is on docker."
* I had not realized the group was significant
* I think we can leave tag pattern as in the recommended (top) one, because the `v*` means "match 0 or more `v` characters. Therefore entering a `v` or not will make no difference. This is more maintainable should we ever switch to using `v`-prefixed versions.
* We can still override to any value by setting a different value in workflow dispatch (there is no regex applied to that value).

I'll make a PR, but will include the extra logging I was planning, because it helps to make the workflow runs less opaque.
